openapi_python_client/__init__.py

```diff
@@ -22,7 +22,12 @@
 from .parser.errors import ErrorLevel, GeneratorError
 from .parser.properties import LiteralEnumProperty
 
-__version__ = version(__package__)
+#__version__ = version(__package__)
+# Benchling-specific change: the original line above won't work because the installed package name of
+# our fork (benchling-openapi-python-client) deliberately does not match the base module name (see
+# pyproject.toml)
+INSTALLED_PACKAGE = "benchling-openapi-python-client"
+__version__ = version(INSTALLED_PACKAGE)
```

Comment:
As noted in a comment I added to `pyproject.toml`, we want our installable package to be called `benchling-openapi-python-client`, but we deliberately did not change the base module name to `benchling_openapi_python_client`, because for our own internal purposes we want to be able to treat this as a drop-in replacement for the upstream package. That means `version(__package__)` will not work because `__package__` actually returns the base module name in this scenario, but `version` wants the installed package name.